github4s/src/main/scala/github4s/Decoders.scala

```diff
@@ -255,4 +247,35 @@ object Decoders {
     }
   }
```

Comment:
Maybe it's better not to use reserved words to avoid using `` 

https://github.com/47deg/github4s/blob/146733dda87ad65582b7e5ef0ae013aebb77b552/github4s/src/main/scala/github4s/Decoders.scala#L154

Reply:
This unfortunate naming is a result of trying to exactly match the names of the JSON fields in the responses.